### DISCUSSION TITLE
Select Entries - further restrict by entry.type

### DISCUSSION THREAD

**Discussion by CreateSean** [solspace/craft-freeform#384] [Feature Requests]
In the select options to choose entries I'd like to further filter by entry.type.

My pages channel has 100+ entries but needs the select to include the 5 entries in the **lodging** entry type. - more could be added in the future, but it should only show from this entry type.

Would like to see additional filter below target - **Entry Type**

![image](https://user-images.githubusercontent.com/6899490/176028339-28c65604-ed38-45d8-8f01-b7f256ed3247.png)

**Comment by kjmartens** [comment]
This has been implemented in Freeform 5. 🙂
